Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,268 @@
+its own coordinate system but there is only one world coordinate system to
+define the position and orientation of each model.
+
+### Anatomical coordinate system
+
+The most important model coordinate system for medical imaging techniques is
+the anatomical space (also called patient coordinate system). This space
+consists of three planes to describe the standard anatomical position of a
+human:
+
+- the *axial plane* is parallel to the ground and separates the head
+  (Superior) from the feet (Inferior).
+- the *coronal plane* is perpendicular to the ground and separates the front
+  from (Anterior) the back (Posterior).
+- the *sagittal plane* separates the Left from the Right.
+
+From these planes it follows that all axes have their notation in a positive
+direction (e.g. the negative Superior axis is represented by the Inferior axis).
+
+The anatomical coordinate system is a continuous three-dimensional space in
+which an image has been sampled. In neuroimaging, it is common to define this
+space with respect to the human whose brain is being scanned. Hence, the 3D
+basis is defined along the anatomical axes of anterior-posterior,
+inferior-superior, and left-right.
+
+However different medical applications use different definitions of this 3D
+basis. Most common are the following bases:
+
+- LPS (Left, Posterior, Superior) is used in DICOM images
+
+$$
+LPS = \begin{Bmatrix}
+\text{from right towards left} \\
+\text{from anterior towards posterior} \\
+\text{from inferior towards superior}
+\end{Bmatrix}
+$$
+
+- RAS (Right, Anterior, Superior) is similar to LPS with the first two axes
+  flipped
+
+$$
+RAS = \begin{Bmatrix}
+\text{from left towards right} \\
+\text{from posterior towards anterior} \\
+\text{from inferior towards superior}
+\end{Bmatrix}
+$$
+
+Thus, the only difference between the two conventions is that the sign of the
+first two coordinates is inverted.
+
+Both bases are equally useful and logical. It is just necessary to know to
+which basis an image is referenced.
+
+### Image coordinate system
+
+The image coordinate system describes how an image was acquired with respect
+to the anatomy. Medical scanners create regular, rectangular arrays of points
+and cells which start at the upper left corner. The $i$ axis increases to the
+right, the $j$ axis to the bottom and the $k$ axis backwards.
+
+In addition to the intensity value of each voxel $(i j k)$ the origin and
+spacing of the anatomical coordinates are stored too.
+
+- The origin represents the position of the first voxel (0, 0, 0) in the
+  anatomical coordinate system, e.g. (100, 50, -25) mm.
+- The spacing specifies the distance between voxels along each axis, e.g.
+  (1.5, 0.5, 0.5) mm.
+
+The following 2D example shows the meaning of origin and spacing:
+
+![image_coordinates](https://github.com/Slicer/Slicer/releases/download/docs-resources/image_coordinates.png){.bg-warning w=300px align=center}
```

Comment:
```suggestion
![image_coordinates](https://github.com/Slicer/Slicer/releases/download/docs-resources/coordinate_systems_image_coordinates.png){w=300px align=center}
```

All images specific to this page have been prefixed with `coordinate_systems_`

Reply:
Done in [0b7a79e](https://github.com/Slicer/Slicer/commit/0b7a79ed1b791335814ae637ee267ba8edf3750b).